db/seeds.rb

```diff
@@ -1,76 +1,38 @@
-# Cases we should cover with seeds
-# - User with no resources, no lists, no groups
-# - User with no resources, one list, no groups
-# - User with resource, list, group
-# - User with one group, no list
-# - Group with all users, one list
-# - Group with 2 users, no list
-# - Resource belongs to a user, and two lists
-# - Resource belongs to no user, and two lists
-# - Resource belongs to one list
-# - User's list
-# - User's list
-# - Group's list
-
-# Create users
-5.times do |n|
-  User.create(
-    email: "user#{n}@greencommons.org",
-    password: 'thecommons'
-  )
+# config
+users_count = 20
+resources_count = 400
+groups_count = 10
+
+p "Creating #{users_count} users..."
+users_count.times do |n|
+  FactoryGirl.create(:user, email: "user#{n}@greencommons.org",
```

Comment:
What about using FactoryGirl's `create_list` for this one. No need to specify email or password, I'm fine with those being randomized values from Faker generated by the factory itself. What do you think?